sync2jira/upstream_issue.py

```diff
@@ -257,26 +311,30 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
-        if not issue.get('storypoints', None):
-            issue['storypoints'] = ''
-            orgname, reponame = upstream.rsplit('/', 1)
-            issuenumber = issue['number']
-            default_github_project_fields = config['sync2jira']['default_github_project_fields']
-            project_github_project_fields = config['sync2jira']['map']['github'].get(upstream, {}).get('github_project_fields', {})
-            github_project_fields = default_github_project_fields | project_github_project_fields
-            variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
-            for fieldname, values in github_project_fields.items():
-                ghfieldname, _ = values
-                variables['ghfieldname'] = ghfieldname
-                response = requests.post(graphqlurl, headers=headers, json={"query": project_items_query, "variables": variables})
-                data = response.json()
-                if fieldname == 'storypoints':
-                    try:
-                        issue[fieldname] = data['data']['repository']['issue']['projectItems']['nodes'][0]['fieldValueByName']['number']
-                    except (TypeError, KeyError) as err:
-                        log.debug("Error fetching %s!r from GitHub %s/%s#%s: %s",
-                                  ghfieldname, orgname, reponame, issuenumber, err)
-                        continue
+        issue['storypoints'] = None
+        issue['priority'] = ''
+        orgname, reponame = upstream.rsplit('/', 1)
+        issuenumber = issue['number']
+        github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
+        variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
+        response = get_github_project_details(graphqlurl, headers, ghquery, variables)
+        if response.status_code != 200:
+            log.debug("Error fetching from GitHub: %s", response.text)
+            continue
+        data = response.json()
+        try:
+            nodes = data['data']['repository']['issue']['projectItems']['nodes'][0]['fieldValues']['nodes']
+            for item in nodes:
+                if 'fieldName' in item:
+                    gh_field_name = item.get('fieldName', {}).get('name')
```

Comment:
Given that, if execution reaches line 329, we know that `'fieldName'` is in `item`, thanks to the `if` statement at line 328, we don't need the first `get()` call:
```suggestion
                if 'fieldName' in item:
                    gh_field_name = item['fieldName'].get('name')
```

Reply:
Fixed in 725a04868080b4dd7aebced4a40a6b28bb8725e5